docroot/themes/humsci/humsci_basic/src/scss/objects/_layouts.general.scss

```diff
@@ -35,6 +35,7 @@
 .ptype-hs-carousel,
 .field-hs-page-hero,
 .ptype-hs-row,
+.ptype-hs-timeline,
```

Comment:
I'm not seeing the same issue with the margin-top when a timeline is in a row with a well. This class _should_ be taking care of that. Let me know if you still see the issue so we can pair and I can see what you see in your local environment. 💃 

<img width="923" alt="well" src="https://user-images.githubusercontent.com/12678977/113075912-ebfb3780-919b-11eb-92c7-75584d37ffe3.png">

Reply:
Yep, I think this was browser caching the padding on this, it is indeed fine. Thank you Heather!